exhale/graph.py

```diff
@@ -2021,7 +1993,7 @@ class view hierarchy (<a href="..."> for the ``createTreeView = True`` option).
             else:
                 path = node.name
 
-            html_safe_name = path.replace(os.sep, "_")
+            html_safe_name = path.replace(":", "_").replace(os.sep, "_").replace(" ", "_")
```

Comment:
Note to self: make an `html_safe_name` method.  I think this happens elsewhere in the code but need to check.

Reply:
yeah I was thinking about that as well. I looked up at benchmarks regarding multiple string replaces as well and it looks like chaining ``.replace(...)`` calls is the most efficient way to do so when there are only a few chars to replace!